cmd/fyne/package.go

```diff
@@ -379,8 +379,12 @@ func (p *packager) doPackage() error {
 		return p.packageLinux()
 	case "windows":
 		return p.packageWindows()
+	case "android/arm":
+		return p.packageAndroid(p.os)
+	case "android/386":
+		return p.packageAndroid(p.os)
 	case "android":
```

Comment:
Also, I think it would be helpful to update flag's description with new cases in `packager.addFlags()`

Reply:
I hope this is what you were thinking.

Reply:
Yes, thank you!